web/models/meeting.ex

```diff
@@ -1,19 +1,54 @@
 defmodule Meetings.Meeting do
-  defstruct title: "", date: DateTime.utc_now(), duration: 0, location: "", description: "", email: "", agenda: ""
+  defstruct title: "", location: "", date: DateTime.utc_now(), duration: 0
 
+  alias Meetings.MeetingType
+  alias Meetings.MeetingDate
```

Comment:
It's because of the discrepancy between the database and the Meeting model, which is meant to be an aggregate. The MeetingType, MeetingDate and MeetingExtra are models for the db tables. Meeting is not. But Meeting needs to be able to generate a list of dates with title, location and duration or whatever fields we need, which can come from all three tables.